action.yml

```diff
@@ -25,6 +25,14 @@ inputs:
     description: 'The branch, tag, or SHA of the repository to clone'
     required: false
 
+  checkout-repository:
+    description: 'The repository to checkout if not the repository that triggered the action. For use when building GitHub Apps'
+    required: false
```

Comment:
default value probably will be needed as `${{ github.repository }}`

Reply:
will update tomorrow after testing.

Reply:
Updated with this default